Standards/CCSMB-13.md

```diff
@@ -0,0 +1,47 @@
+# CCSMB-13: Stereo DFPWM Audio Protocol
+
+*Author: @Drucifer (Discord)*
+
+*Version: 0.1*
+
+*Last revised: YYYY-MM-DD*
+
+## Quick information
+
+| Information |                             |
+| ----------- | --------------------------- |
+| Version     | 0.1                         |
+| Author      | @Drucifer (Discord)         |
+| Type        | Stereo DFPWM Audio Protocol |
+| MIME        | `audio/sdap`                |
+
+## Introduction
+
+This standard serves to provide framework for unifying and informing the layout of transmissions of stereo DFPWM content on modem channels to increase access to the general public while preserving the hobbyist usage.
+
+## Technical details
+
+Stereo DFPWM Audio Protocol (SDAP) has two functional rulesets depending on the modem used:
+| Modem Type | Transmission Mode |
+| ---------- | ----------------- |
+| Ender      | Full-Power        |
+| Wireless   | Low-Power         |
+| Wired      | Low-Power/Wired   |
+
+
+*Important: SDAP does not use Rednet! SDAP is a direct-modem protocol! Broadcasting SDAP packets via Rednet on a public server may result in bad things happening and I won't be held responsbile. Always make sure you're using direct modem calls, not Rednet!*
+
+Common to all modes of transmission (Full/Low-Power & Wired), modem transmissions are sent to the assigned channel, using the modem return channel id as the PID value, the packet formatting shall be pack formatted `s1s1s2`: First string (max 255 characters) containing the station callsign immediately followed by the station's full printed name (e.g. `KDRU Drucifer's Private Reserves`) and shall not be altered mid-operation (if you need to change it, shut the station down first, then restart it with the new name in place), second string (max 255 characters) contains the current program title (typically `Artist - Song`) and may be altered on every packet transmitted, third string contains the stereo DFPWM payload (expected to be 12000 bytes per second total, 6000 bytes per channel, left channel first, though you may experiment with alternate packet sizes and timing using Low-Powered and Wired setups, see Low-Power Transmission). This protocol is intended to be broadcast only by transmitters, and received by N tuners in a uni-directional exchange, no bi-directionality given.
+
+Example modem transmit snippet to give you an idea of where things go: `modem.transmit(65500,1337,string.pack("s1s1s2","KDRU Drucifer's Private Reserves","Artist - Title",leftChannel:sub(1,6000)..rightChannel:sub(1,6000))` where `leftChannel` and `rightChannel` contain DFPWM encoded aduio buffers of 6000 or more bytes.
```

Comment:
This could probably better be a full codeblock instead of an inline codeblock

Reply:
Added 2, count 'em, two codeblocks with expounded code samples.